tmva/tmva/src/DecisionTree.cxx

```diff
@@ -918,6 +1279,100 @@ void TMVA::DecisionTree::GetRandomisedVariables(Bool_t *useVariable, UInt_t *map
    if (nSelectedVars != useNvars) { std::cout << "Bug in TrainNode - GetRandisedVariables()... sorry" << std::endl; std::exit(1);}
 }
 
+// Multithreaded version of DecisionTree::TrainNodeFast
+#ifdef R__USE_IMT
+//====================================================================================
+// Had to define this struct to enable parallelization in TrainNodeFast.
```

Comment:
This comment should be rewritten to be more general (describe intent of the struct) and using doxygen documentation style.